### PR TITLE
updated dependencies, fixes #10

### LINKED ISSUE #10: outdated dependencies
(AUTH-4123)
[01-Authorization sample](https://github.com/auth0-samples/auth0-aspnet-owin-webapi-samples/tree/master/Quickstart/01-Authorization/WebApi) uses outdated version of packages such as IdentityModel, Newtonsoft.Json, etc

### PR DESCRIPTION
**Pull request by Amialc**

### CODE REVIEW

**Comment by jerriep** [comment]
@Amialc 

Did you test this? I remember from previously that version 5.x of that library did not work with OWIN because it is supposed to work with ASP.NET Core only.

Please double check this for me

**Comment by rubiktubik** [comment]
I've tried upgrading to 5.x on ASP.NET 4 and it not worked!

File: Quickstart/01-Authorization/WebApi/WebApi/Web.config [hunk 1]
```diff
@@ -5,8 +5,8 @@
   -->
 <configuration>
   <appSettings>
-    <add key="Auth0Domain" value="{DOMAIN}" />
-    <add key="Auth0ApiIdentifier" value="{API_IDENTIFIER}" />
+    <add key="Auth0Domain" value="tutorials.auth0.com" />
```
Comment (jerriep) [inline Quickstart/01-Authorization/WebApi/WebApi/Web.config]: Please revert to `{DOMAIN}`

File: Quickstart/01-Authorization/WebApi/WebApi/Web.config [hunk 2]
```diff
@@ -5,8 +5,8 @@
...
+    <add key="Auth0ApiIdentifier" value="https://api.tutorials.auth0.com" />
```
Comment (jerriep) [inline Quickstart/01-Authorization/WebApi/WebApi/Web.config]: Please revert to `{API_IDENTIFIER}`

**Comment by Amialc** [comment]
@jerriep I've tested, it works for me.

**Pull request merged**